src/Combobox/index.js

```diff
@@ -175,7 +174,9 @@ const Combobox = ({
   }
 
   const [displayedItems, setDisplayedItems] = useState(items);
-  const [clearOnNextInput, setClearOnNextInput] = useState(false);
```

Comment:
no longer needed - input clearing is all handled in the `stateReducer` and `onInputValueChange`